app/uk/gov/hmrc/gform/config/ConfigModule.scala

```diff
@@ -92,7 +90,8 @@ class ConfigModule(val context: ApplicationLoader.Context, playBuiltInsModule: P
       optimizelyUrl = for {
         url       <- playConfiguration.getOptional[String]("optimizely.url")
         projectId <- playConfiguration.getOptional[String]("optimizely.projectId")
-      } yield s"$url$projectId.js"
+      } yield s"$url$projectId.js",
+      hmrctcs = new hmrcTrackingConsentSnippet(new TrackingConsentConfig(context.initialConfiguration))
```

Comment:
Use `playConfiguration` instead of `context.initialConfiguration` (to stay consistent)

Reply:
Perfect. Done.